src/platform/pantherlake/include/platform/trace/trace.h

```diff
@@ -0,0 +1,26 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
this file is identical as in MTL
I wonder how many of those headers can be unified...?

Reply:
I think that we could huge part of it move to DTS and use zephyr configurator.

Reply:
@marcinszkudlinski  @pjdobrowolski This is an ongoing task to remove the remaining src/plafform usages. I think MTL is good reference and if you are not adding more code under platform, that is ok. In the end, we will be removing most of this, but we have some remaining common code that needs to build for both XTOs and Zephyr, and this creates need to have some per-platform definitions (like the trace.h file). So I'd say out-of-scope for this PR.